Pipfile

```diff
@@ -12,9 +12,14 @@ plotly = "*"
 dash_dangerously_set_inner_html = "*"
 pandas = "*"
 beaker = "*"
+scipy = "*"
 
 [dev-packages]
 flask = "*"
+black = "*"
 
 [requires]
 python_version = "3.6"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
Why is this needed?

Reply:
This was needed to allow for Python Black to be installed, as it has yet to have an official "production" release, and only provides pre-releases for its multiple versions. Removing this along with Python Black.